docs/common/install_and_configure_nginx_with_rtmp_module_on_windows.md

```diff
@@ -101,29 +427,31 @@ rtmp {
         application ingest {
             live on;
             record off;
-	    
-	        on_publish http://localhost/api/bot/validate-stream-key;
+        
+            on_publish http://localhost/api/bot/validate-stream-key;
         }
+
         application test-endpoint {
             live on;
             record off;
+        
+            on_publish http://localhost/api/bot/validate-stream-key;
```

Comment:
If we provide an invalid stream key in the RTMP URL, the player will not be able to play the stream because the extraction endpoint does not exist. The RTMP endpoints generated have the following structure:

rtmp://domain.co:port/application-name/stream-key.

Reply:
@verdua If I understand correctly that means that even if we listen to all the ports we are adding here in the NGINX configuration, the request need to match the structure you mentioned above for the connection to be successful, which is why we don't need to go back into the API to check the stream key of the request?

Have we checked what happen if we change the stream key during a meeting? (not during an extraction, but for example by stopping and extraction, regenerating the key and trying to start a new extraction with the new key).

Reply:
@sw-dcherubini We have tested changing the stream key during the meeting and starting a new extraction again and we had no problems consuming the extraction with the new stream key, with the old one it does not work.